pkg/pebinaryexecutor/pebinary.go

```diff
@@ -254,6 +277,15 @@ func getArgs(argsMap map[string]string) []string {
 	return args
 }
 
+func argIndex(args []string, s string) (int, error) {
+	for i, arg := range args {
+		if strings.Contains(arg, s) {
+			return i, nil
+		}
+	}
+	return -1, errors.New("no matching argument found")
```

Comment:
What significance does `-1` have here? Can you create a constant with a descriptive name for this value?

Reply:
What should I return as a default if the value isn't found? What is norm for that?

Reply:
I would return `0`. Anything else implies meaning and that it'd be checked. In this case, I'd assume the `err` will be checked and in that case, the calling code won't be checking the int value.